src/components/Pagination/index.js

```diff
@@ -29,12 +29,12 @@ class Pagination extends React.Component {
   previousPageLink( pager ) {
     return pager.isFirstPage() ?
       <li className="disabled">
-        <a href="#" aria-label="Previous">
+        <a href="#" aria-label="Previous" onClick={ function(e){ e.preventDefault(); } }>
```

Comment:
Being forced to add a `preventDefault` on every `<a>` seems bloated. Can't we just get rid of the `<a>` element ?